configure.ac

```diff
@@ -322,6 +322,22 @@ AC_ARG_ENABLE([fips],
     [ENABLED_FIPS=$enableval],
     [ENABLED_FIPS="no"])
 
+# wolfProvider Options
+AC_ARG_ENABLE([wolfprovider],
+    [AS_HELP_STRING([--enable-wolfprovider],[Enable wolfProvider options (default: disabled)])],
+    [ ENABLED_WOLFPROVIDER=$enableval ],
+    [ ENABLED_WOLFPROVIDER=no ]
+    )
+if test "x$ENABLED_WOLFPROVIDER" != "xno"
+then
+    enable_all_crypto=yes
+    enable_opensslcoexist=yes
+    enable_sha=yes
+    enable_eccminsz=192
+    with_max_ecc_bits=1024
```

Comment:
These should be conditional on the user not supplying an override value to configure.  E.g.
```
    test -z "$enable_all_crypto" && enable_all_crypto=yes
```
Notice that `--enable-all`, `--enable-all-crypto`, and even FIPS `dev` or `v5-dev`, use similar logic to allow tailoring and testing of different feature combinations.  It's understood that some combinations won't work -- if the user forces something, they own the consequences.